### DISCUSSION TITLE
Plot + name units of multiple values in the graph

### DISCUSSION THREAD

**Discussion by JonasDaWi** [technologiestiftung/stadtpuls#15] [Q&A] [answered]
I want to show an array of data (temperature, moist %) via stadtpuls. I know that this is not supported yet, but in the documentation you state, that it is possible to send multiple values in the measurements array. 
The problem is: when I send more than one value in the measurement array, no value is shown on the graph. It only works when I send 1 value.

**Comment by JonasDaWi** [comment]


**Reply by ff6347** [reply]
Can you show us your formatter function and also the full error message?

**Reply by ff6347** [reply]
Plus. Which sensor are we talking about? Can you give us the ID. 

Thanks.
FYI me investigating this issue might take some time. It's Friday and I'm actually on vacation. 🏖

**Reply by JonasDaWi** [reply]
Thanks for the quick reply. It seems to work now! My sensor id is 77
In case anyone is interested, this is my upload method:
```
function decodeUplink(input) {
  var bat = ((input.bytes[0] << 8) + input.bytes[1])/1000;
  var moist = ((input.bytes[4] << 8) + input.bytes[5])/100;
  
  var temp = 0.00;
  if ((((input.bytes[6] << 8) + input.bytes[7]) & 0x8000)>>15 === 0){
    temp = ((input.bytes[6] << 8) + input.bytes[7]) /100;
  }
  if ((((input.bytes[6] << 8) + input.bytes[7]) & 0x8000)>>15 === 1){
    temp = (((input.bytes[6] << 8) + input.bytes[7])-0xFFFF)/100;
  }
  
  var conduct = (input.bytes[8] << 8) + input.bytes[9];
  
  return {
    data: {
      bytes: input.bytes,
      measurements: [ moist, temp, bat, conduct ]
    },
    warnings: [],
    errors: []
  };
}
```

It is a Dragino LSE01 Sensor.

So my request would be: Please add the ability to plot multiple values in a graph and to name the unit of the measurements.

**Comment by Esshahn** [comment, marked as answer]
Cool that it works now and thank you for sharing the info. The ability to plot multiple values on a graph is already in our backlog, so we're covering that soon.
